apps/studio/pages/project/[ref]/logs/collections/[collectionToken].tsx

```diff
@@ -0,0 +1,10 @@
+import { WarehouseCollectionDetail } from 'components/interfaces/DataWarehouse/WarehouseCollectionDetail'
+import { LogsLayout } from 'components/layouts'
+import { observer } from 'mobx-react-lite'
+import { NextPageWithLayout } from 'types'
+
+const PageLayout: NextPageWithLayout = () => <WarehouseCollectionDetail />
+
+PageLayout.getLayout = (page) => <LogsLayout>{page}</LogsLayout>
+
+export default observer(PageLayout)
```

Comment:
no need for observer - observer is only for if we're using mobx (which we're deprecating)